nifi-framework-bundle/nifi-framework-extensions/nifi-py4j-framework-bundle/nifi-python-framework/src/main/python/framework/ExtensionManager.py

```diff
@@ -285,37 +285,31 @@ def import_external_dependencies(self, processor_details, work_dir):
             logger.info("All dependencies have already been imported for {0}".format(class_name))
             return True
 
-        python_cmd = os.getenv("PYTHON_CMD")
+        packages_to_install = []
```

Comment:
Recommend renaming this variable as it includes both packages and `requirements.txt` when found.
```suggestion
        dependency_references = []
```

Reply:
I have renamed it in c32d56bfb247e80fb072c062ea6b1e32d1a4f3d2